### PR TITLE
[12.0][ADD] hr_timesheet_sheet_no_create

### PR DESCRIPTION
**Pull request by astirpe**
~This change was asked by our customers.~

With this module it will not be possible to create projects and tasks
directly from the lines of a timesheet sheet.

### CODE REVIEW

**Comment by alexey-pelykh** [comment]
That's a bit corner case, we solved similar issue with project_role using other module from server_ux, web_m2x_options. E.g. we're using task creation from the timesheets :(

**Comment by MiquelRForgeFlow** [comment]
My opinion (and our company opinion) is similar to what @alexey-pelykh said. We think it's not something to be "fixed", but just an optional customization that should be handled in another module.

**Comment by astirpe** [comment]
Yes, we already use a separate module. But since it's such a simple module, would it be well accepted  in case I propose it here?

**Comment by MiquelRForgeFlow** [comment]
> But since it's such a simple module, would it be well accepted in case I propose it here?

Of course we would accept that simple module here. Go ahead.

File: hr_timesheet_sheet_no_create/views/hr_timesheet_sheet.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
```
Comment (MiquelRForgeFlow) [inline hr_timesheet_sheet_no_create/views/hr_timesheet_sheet.xml:5]: you can add if you want

```suggestion
<!-- Copyright <YEAR(S)> <AUTHOR(S)>
     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
<odoo>
```

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

**Review by alexey-pelykh — APPROVED** [review]
Code LGTM

**Pull request merged**